packages/components/src/styles/components/side-nav/vars.scss

```diff
@@ -19,7 +19,7 @@
   // breakpoint
   --hds-app-desktop-breakpoint: #{map.get($hds-breakpoints, "lg")}; // this is used only to read its value via JS and set the `hds-side-nav--is-desktop` class
   // widths
-  --hds-app-sidenav-width-minimized: 48px;
+  --hds-app-sidenav-width-minimized: 16px;
```

Comment:
It may take some time for all product teams to update from the old `SideNav` to the new `AppSideNav`. Changing this value here will mess up the layout in the old SideNav as it will be too narrow for the HomeLink logo to fit.

Would it be possible to update the related value for your sticky footer and other components instead?